packages/playground/website/public/logger.php

```diff
@@ -33,13 +33,13 @@ function response($ok, $error = null)
 function validate_message($message)
 {
     // Validate description. Description is required
```

Comment:
Wouldn't it be easier if the data was transmitted not as a text blob but as separate POST fields? `$_POST['description']`, `$_POST['logs']` etc?

Reply:
I was thinking the same things while working on the validation but ended up overcomplicating it.